provider/virt_storage/backend/ceph.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from provider.virt_storage.helper import rbdcli
+
+from provider.virt_storage import storage_volume
+from provider.virt_storage.backend import base
+
+
+class CephPool(base.BaseStoragePool):
+    TYPE = "rbd"
+
+    def __init__(self, name):
+        self.image = None
+        self.server = None
+        super(CephPool, self).__init__(name)
+
+    @property
+    def helper(self):
+        if self._helper is None:
+            self._helper = rbdcli.RBDCli(self.target.path)
```

Comment:
For rbd, we use rbd pool name(and namespace if it's supported in future) to describe the rbd backend, so here:
self._helper = rbdcli.RBDCli(self.source.pool_name)
why pool name is set in source? please refer to 'Example pool input' on page https://libvirt.org/storage.html#StorageBackendRBD

Reply:
Have updated, thanks zhenchao.